### PR TITLE
navbar with icons

### PR DESCRIPTION
**Pull request by stshayna**
1. Created basic navbar with 4 icons and a dropdown menu for login/log out.
Details, e.g. icons/links, can change after any further discussion!

2. Updated colors of messages in chatroom to align with Figma.

Screenshot 2022-05-26 at 11 34 32 PM
Screenshot 2022-05-26 at 11 34 48 PM
Screenshot 2022-05-26 at 11 35 16 PM
Screenshot 2022-05-26 at 11 35 28 PM

### CODE REVIEW

**Comment by MarlAndre** [comment]
1. We changed the first icon for the map marker instead of the home, so it will be clearer for the user what is the purpose of that icon
2.  I think having the avatar image in the navbar is a bit weird, I would change it for an icon so we have the same continuity in the navbar

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,39 +1,32 @@
-<%# this is a boilerplate %>
-<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+<div class="navbar fixed-bottom navbar-expand-sm navbar-light navbar-lewagon py-3">
+  <%# 4 icons + one collapse button in the navbar %>
   <div class="container-fluid">
-    <%= link_to "#", class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-    <% end %>
+    <a href="/"><i class="fas fa-home"></i></a>
```
Comment (MarlAndre) [inline app/views/shared/_navbar.html.erb:4]: Change home icon for map marker icon

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,39 +1,32 @@
...
+    <a href="/items"><i class="fas fa-search"></i></a>
+    <a href="/items/new"><i class="far fa-plus-square"></i></a>
+    <a href="/chatrooms"><i class="far fa-comment"></i></a>
 
+    <%# collapse button shows the avatar image (for the demo user only) or a normal icon (for everyone else) %>
     <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
+      <% if current_user == User.first %>
+        <%= image_tag "demo_user.png", class: "avatar dropdown-toggle text-white", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```
Comment (MarlAndre) [inline app/views/shared/_navbar.html.erb:12]: Having an image next to a bunch of icons is weird, I would put the profile icon back

**Review by MarlAndre — CHANGES_REQUESTED** [review]

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -1,39 +1,32 @@
...
-              <%= image_tag "demo_user.png", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
-            <% else %>
-              <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
-            <% end %>
-            <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
-              <%= link_to "Action", "#", class: "dropdown-item" %>
-              <%= link_to "Another action", "#", class: "dropdown-item" %>
-              <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item" %>
-            </div>
+            <%= link_to "Log out", destroy_user_session_path, method: :delete %>
```
Comment (MarlAndre) [inline app/views/shared/_navbar.html.erb:25]: If we want to login / logout, it would make more sense to do it by clicking on the profile icon

**Review by MarlAndre — CHANGES_REQUESTED** [review]

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (stshayna) [reply app/views/shared/_navbar.html.erb]: changed the icon!

File: app/views/shared/_navbar.html.erb (hunk 2 above)
Comment (stshayna) [reply app/views/shared/_navbar.html.erb]: ok, removed the profile picture as requested

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (stshayna) [reply app/views/shared/_navbar.html.erb:21]: ok, removed the profile picture as requested

**Comment by stshayna** [comment]
Navbar after changes:

Screenshot 2022-05-27 at 11 51 56 AM
Screenshot 2022-05-27 at 11 52 08 AM

**Comment by stshayna** [comment]
Navbar after changes:
1. removed profile picture
2. changed home icon to map marker icon
3. moved the 'Login"/"Log out" text to the end
Screenshot 2022-05-27 at 11 51 56 AM
Screenshot 2022-05-27 at 11 56 06 AM

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (MarlAndre) [reply app/views/shared/_navbar.html.erb:21]: So, if we click on the profile icon, there's a dropdown to logout ? But what if the user want to go to heir profile page, is there a route for that or we will create it later ?

Comment (stshayna) [reply app/views/shared/_navbar.html.erb:21]: There can be two ways, we can:
1. Add "Profile Page" above "Log out" in the dropdown menu; or
2. Remove the dropdown menu and replace the url by user profile page

I'm fine with both options!

Comment (MarlAndre) [reply app/views/shared/_navbar.html.erb:21]: Ok, let's talk about it tomorrow. I'll merge the branch :)

**Pull request merged**